src/SecurityChecker/Command/SecurityCheckerCommand.php

```diff
@@ -1,12 +1,157 @@
 <?php
-declare(strict_types=1);
+declare(strict_types = 1);
+
 /**
  * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
  * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
  */
-namespace Brancho\SecurityChecker;
 
-class SecurityCheckerCommand
+namespace SecurityChecker\Command;
+
+use SecurityChecker\Result;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\Process;
+
+class SecurityCheckerCommand extends Command
 {
+    protected const COMMAND_NAME = 'security:check';
+
+    protected const BINARY_CHECKER = 'https://github.com/fabpot/local-php-security-checker/releases/download/v1.0.0/local-php-security-checker_1.0.0_linux_amd64';
+    protected const FILE_NAME = '/tmp/security-checker';
+    protected const FALSE_POSITIVE_ISSUE_NUMBER = 'CVE-NONE-0001';
+
+    /**
+     * @return void
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setName(static::COMMAND_NAME)
+            ->setDescription('Checks security issues in your project dependencies');
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return int
+     */
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->loadFile();
+        $this->changeFileMode();
+        $commandOutput = $this->runCommand();
+        $commandOutput = $this->markFalsePositiveResults($commandOutput);
+
+        $result = $this->createResultFromCommandOutput($commandOutput);
+
+        $output->writeln($result->getVulnerabilities());
+
+        return $this->convertResultToExitCode($result, $output);
+    }
+
+    /**
+     * @return void
+     */
+    protected function loadFile(): void
+    {
+        $process = new Process(['wget', static::BINARY_CHECKER, '-O', static::FILE_NAME]);
```

Comment:
I would check file existance first. As I can see we don't remove file after the script execution, so it's might be possible that it will be in the `temp` folder on the next execution

Reply:
Yes, but mostly this code will run on travis, where it will be removed any times. 
should i check if this file exist, or just delete it after execution?

Reply:
check if exists and dont re-download.